src/get_usernames.py

```diff
@@ -0,0 +1,83 @@
+#!/usr/bin/env python3
+"""Gets the usernames of students taking courses this semester"""
+
+__authors__    = ["Erik Vesteraas"]
+__copyright__  = "Erik Vesteraas"
+__license__    = "MIT"
+# This file is subject to the terms and conditions defined in
+# file 'LICENSE.txt', which is part of this source code package.
+
+import paramiko
+import getpass
+import argparse
+import sys
+import os
+from datetime import date, timedelta
+from file_funcs import load_json
+
+def get_args():
+    argparser = argparse.ArgumentParser(description='Get usernames of students taking courses this semester')
+    argparser.add_argument('--username', '-u', help='Username for login', type=str)
+    argparser.add_argument('--password', '-p', help='Password for login', type=str)
+    argparser.add_argument('--prev', help='Read data for the previous semester', action='store_true')
+    args = argparser.parse_args()
```

Comment:
Would be useful to have an optional argument `--filename` or `--input` if you want to specify a different course list than `spring.json` or `fall.json`